CHANGELOG.md

```diff
@@ -5,53 +5,223 @@ All notable changes to this project will be documented in this file.
+  - Filter custom buttons according to visibility expressions [(#1824)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1824)
+  - Fix icon for Ansible credential add button [(#1687)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1687)
+  - Dialog editor fixes [(#1728)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1728)
+  - Fixed issue with custom button editing [(#1686)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1686)
+  - Custom button submit all [(#1572)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1572)
+  - Add status to Repository Properties [(#1131)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1131)
+  - Add button classes [(#2371)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2371)
+- Chargeback
+  - UI Fix for ChargeableField in chargeback editor [(#1727)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1727)
 - Compute
   - Add policy buttons to physical servers page [(#2284)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2284)
   - Group by docker label in ChargebackContainerImage [(#2277)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2277)
   - Adds configuration buttons to physical server toolbar [(#2263)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2263)
   - Use quota_display in Container Project summary screen [(#2090)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2090)
+  - Container Provider: change authentication status labels [(#2104)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2104)
+  - Set the metrics auth status to the metrics tab [(#2102)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2102)
+  - Consider archived nodes for single provider dashboard [(#2084)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2084)
+  - Use endpoint's name instead of role [(#2080)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2080)
+  - Hide Archived Container Nodes [(#1524)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1524)
+  - Add link to network manager for infra providers [(#2028)](https://github.com/ManageIQ/manageiq-ui-classic/pull/2028)
+  - Quad icons for containers providers [(#1950)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1950)
+  - Add support for displaying network adapter details [(#1654)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1654)
+  - Add support for monitoring selection in UI [(#1501)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1501)
+  - Add alerts to dashboard [(#1234)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1234)
+  - Create a box in new dialog automatically [(#1791)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1791)
+  - Generic Object Instances UI - `show_list` and `show` [(#1765)](https://github.com/ManageIQ/manageiq-ui-classic/pull/1765)
+  - Use the fonticon picker component to pick custom button icons [(#1801)](https://github.com/ManageIQ/manageiq-ui-classic)
```

Comment:
Still wrong... `https://github.com/ManageIQ/manageiq-ui-classic/pull/1801` 😄

Reply:
Ugh! That's what I get for doing it late at night. Fix coming up. and thanks @simaishi 😃